requirements.txt

```diff
@@ -1,4 +1,4 @@
 Django>=2.2.10
```

Comment:
I believe this is quite an insecure version of Django now, so it's worth investigating how easy it would be to use the latest version.

Reply:
I've updated music21 a few times - but even that is a year or so old now.  But yes - Django must be very out of date!

Reply:
Fixed in a3e987b
Upgrading Django required upgrading Python - which meant upgrading everything else...
Possibly could have upgraded Python a bit further?
I was expecting lots of code to update for talkingscores - but very few changes were needed :-)